### DISCUSSION TITLE
React Native Web support

### DISCUSSION THREAD

**Discussion by haveamission** [invertase/react-native-firebase#4558] [Ideas] [answered]
Is there any possibility of React Native Web support?

I'd like to use Facebook and Google authentication via Firebase with React Native Web, but can't find any way to do this

**Comment by mikehardy** [comment, marked as answer]
This is not on our roadmap right now but if someone was motivated and posted up a strategy that seemed to work and did a PR, I can't think of a reason we wouldn't merge it. As it is we do our best to have our API shape match firebase-js-sdk so it doesn't seem so far-fetched. So far no one has offered to contribute though

**Comment by haveamission** [comment]
I'll take a look into it, though no promises at this point. I need to integrate Firebase into a monorepo, so it might be worth the time to make a PR.

Let me think on it.
